prisma/schema.prisma

```diff
@@ -0,0 +1,140 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+// TODO: Should we unique email? What if two people put the same one but have different FC accounts?
+// TODO: Cache farcaster account metadata? Probably fine to just pull on login/token validation?
+// TODO: Do we need to handle deletion of accounts? (This is a GDPR thing, right?)
```

Comment:
1. email is for contract purposes only, don't think it's necessary to enforce it being unique
2. -
3. yes